misc/sample_application/image_builder/create_image.sh

```diff
@@ -91,6 +91,17 @@ function copy_and_enlarge_image() {
         growpart ${dest_image} 1
 }
 
+function dump_proxy() {
+	local temp_file=$(mktemp /tmp/proxy.XXXX)
+
+	sudo apt-config dump | grep -i proxy > ${temp_file} 2>&1
+	sudo mv ${temp_file} proxy.conf
+
+	echo "$(env | grep -E 'http_proxy|HTTP_PROXY' | sed -e 's/^/export /')" > bashrc
+	echo "$(env | grep -E 'https_proxy|HTTPS_PROXY' | sed -e 's/^/export /')" >> bashrc
```

Comment:
Line 100 will also find what line 101 finds.

You can simply use one line that searches for any occurrence of `_proxy` (ignoring cases). This will also make other variables such as `no_proxy`  to the guest.

Reply:
@junjiemao1  modify the grep command parameter and force update the PR.